esx_service/utils/auth_api.py

```diff
@@ -190,7 +190,8 @@ def modify_privileges(privileges, allow_create, volume_maxsize_in_MB, volume_tot
     logging.debug("modify_privileges: allow_create=%s, volume_maxsize_in_MB=%s, volume_totalsize_in_MB=%s",
                   allow_create, volume_maxsize_in_MB, volume_totalsize_in_MB)
     if allow_create is not None:
-        if allow_create == "True":
+        allow_create = str(allow_create)
```

Comment:
oh wait! this doesn't solve the problem fully. 

auth_api._tenant_access_set() just checks for ```true``` and fails to filter out ```Invalid``` as mentioned the testcase (vmdkops_admin_test.py#739) falls to ```false``` rather than complaining.